consensus/byzantine_test.go

```diff
@@ -464,9 +464,9 @@ func TestByzantineConflictingProposalsWithPartition(t *testing.T) {
 	select {
 	case <-done:
 	case <-tick.C:
-		for i, reactor := range reactors {
+		for i := range reactors {
 			t.Logf(fmt.Sprintf("Consensus Reactor %v", i))
-			t.Logf(fmt.Sprintf("%v", reactor))
+			t.Logf(fmt.Sprintf("%v", css[i].GetRoundState()))
```

Comment:
[optional] the lack of context makes this variable seem difficult to use for debugging

```suggestion
			t.Logf(fmt.Sprintf("round state: %v", css[i].GetRoundState()))
```